src/Encamina.Enmarcha.SemanticKernel.Abstractions/Events/MemoryStoreEventTypes.cs

```diff
@@ -1,32 +1,27 @@
 ﻿namespace Encamina.Enmarcha.SemanticKernel.Abstractions.Events;
 
 /// <summary>
-/// Represents the specific type of operation from a <see cref="IMemoryManager"/>.
+/// Represents the specific type of operation from a <see cref="IMemoryStoreExtender"/>.
 /// </summary>
-public enum MemoryManagerEventTypes
+public enum MemoryStoreEventTypes
 {
     /// <summary>
-    /// 'Undefined' event type.
+    /// 'CreateCollection' event type.
     /// </summary>
-    Undefined = 0,
+    CreateCollection = 0,
 
     /// <summary>
-    /// 'Get' event type.
+    /// 'DeleteCollection' event type.
     /// </summary>
-    Get = 1,
```

Comment:
I would re-add the `Get` and `UpsertBatch` events here and in the `MemoryStoreExtender` implementations. I think that differentiated events are valuable.

Reply:
Thank you for your contribution. You are right. I will update it.